projects/biometrics/index.md

```diff
@@ -0,0 +1,121 @@
+the use of facial recognition technology. LPL hosts webinars with community
+partners to educate about:
+
+ - The alarming ways in which the Chicago Police Department uses technology (e.g.,
+facial recognition technology, biometric technology, social media monitoring,
+and license plate monitoring) for surveillance purposes;
+- The potential consequences that may occur if politicians do not soon pass moratoriums; and
+- The impact of biometric surveillance on our rights to privacy, anonymity, and
+free expression.  
+
+## Advocacy and Legal Action
+
+Together, these community organizations protect constituents, educate community
+members, dismantle utilization of facial recognition and biometric technology,
+and enhance transparency about how police departments use these software
+programs. By uniting their power, the union also educates politicians by writing
+letters, publishing opposite the editorial pages, and meeting with members of
+the Chicago City Council. The alliance also lobbies and [converses
+with](https://www.aclu-il.org/en/candidate-answers-facial-recognition)
+policymakers to empower suspension of facial recognition technology expansion
+amendments.
+
+In a resounding victory, the Illinois Supreme Court [ruled in
+favor](https://lucyparsonslabs.com/posts/BIPA-Victory/)  of a brief LPL joined
+in defense of  the Illinois’ Biometric Privacy Act, the strongest privacy law of
+its kind in the country.  The Privacy Act prohibits the collection and storage
+of facial and fingerprint identifiers of current and former residents of
+Illinois. This landmark ruling is critical because while other cities have
+introduced bills to curb [the use of facial recognition
+technology(https://www.eff.org/aboutface/bans-bills-and-moratoria), a facial
+recognition vendor, —Clearview AI– illicitly collected [three billion
+images](https://chicago.suntimes.com/crime/2020/1/29/21080729/clearview-ai-facial-recognition-chicago-police-cpd)
+scraped from social media and other online sources. Clearview claims not to be
+[constricted](https://www.muckrock.com/foi/atlanta-325/facial-recognition-atlanta-ga-76491/)
+by state biometric privacy laws yet our coalition’s advocacy will allow
+[anyone](https://www.aclu.org/press-releases/aclu-sues-clearview-ai) in Illinois
+to bring monetary claims against Clearview and vendors like it.
+
+LPL continues to drive action to dismantle the utilization of inaccurate face
+geometry data so we may one day live in a world free of discrimination due to
+biometric and facial recognition technology. These goals will prohibit the
+actions of companies from capturing personal data for profit and police who
+collect biometric data through inaccurate racially biased algorithms. Therefore,
+we would live in a safer world as civil liberties will be better protected, and
+police officers will not arrest innocent people for crimes they did not commit
+due to inaccurate algorithms.
```

Comment:
An idea for the last sentence:

"By pressing pause on the disturbing trends in biometric surveillance,
we help create a safer world where civil liberties will be better protected, and
police officers will not arrest innocent people for crimes they did not commit
due to inaccurate algorithms."